web_src/js/components/DiffFileTree.vue

```diff
@@ -5,7 +5,7 @@
   >
     <!-- only render the tree if we're visible. in many cases this is something that doesn't change very often -->
     <div class="ui list">
-      <DiffFileTreeItem v-for="item in fileTree" :key="item.name" :item="item" />
+      <DiffFileTreeItem v-for="item in fileTree" :key="item.name" :item="item" :active-hash="activeHash"/>
```

Comment:
Same here, what about calling it `selectedFile` instead?

Reply:
done!